README.md

```diff
@@ -1,16 +1,192 @@
-# Title
+# Arrow Function Shorthand
+
+## Introduction
+
+The original style for declaring functions in JavaScript is the _function
+declaration_.
+
+```js
+function foo() {
+  return 'bar';
+}
+```
+
+But JavaScript has two other ways to write functions: the _function expression_
+and the _arrow function_. None of these is more correct or better than others.
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Declare a function using a function expression
+- Declare a function using an arrow function
+- Describe situations where arrow functions are used
 
-## Introduction
+## Declare a Function Using a Function Expression
+
+Thus far we've only ever used a _function declaration_:
+
+```js
+function foo() {
+  return 'bar';
+}
+```
+
+A function can also be written:
+
+```js
+let foo = function() {
+  return 'bar';
+}
+```
+
+The `function() {...}` to the right of the assignment operator (`=`) is called
+a _function expression_. The best way to understand what a function expression
+is is by analogy.
+
+```js
+let sum = 1 + 1
+```
+
+Evaluate the expression `1 + 1`, returning `2` and assign it to the variable `sum`
+
+```js
+let difference = 10 - 1;
+```
+
+Evaluate the expression `10 - 1`, returning `9` and assign it to the variable
+`difference`.
+
+```js
+let foo = function() {
+  return 'bar';
+}
+```
+
+Evaluate the expression `function() { return 'bar' }`, returning a thing that
```

Comment:
All of these sentences beginning with "Evaluate ..." confuse me a bit. I would vary and soften these a bit so it's clearer that we're moving from example to example and not instructions (those are later). It would also just improve the flow of the piece.